src/main/java/org/apache/commons/math4/stat/correlation/KendallsCorrelation.java

```diff
@@ -151,7 +155,12 @@ public RealMatrix computeCorrelationMatrix(final double[][] matrix) {
      * @return Returns Kendall's Tau rank correlation coefficient for the two arrays
      * @throws DimensionMismatchException if the arrays lengths do not match
      */
-    public double correlation(final double[] xArray, final double[] yArray)
+    @SuppressWarnings({"lowerbound:array.access.unsafe.low", "index:array.access.unsafe.high.range", "index:array.access.unsafe.high"}) /*
+    #1: i < iEnd < n and j < jEnd < n, hence i and j are @IndexFor(pairs)
+    #2: copyLocation = offset < n, hence copyLocation is @IndexFor("pairDestination")
+    #3: i < n is checked as the loop condition
```

Comment:
None of these comments justifies a lowerbound suppression; they all address the upperbound issues. What lowerbound error was issued, and why?